etcdserver/server.go

```diff
@@ -939,7 +939,7 @@ func (s *EtcdServer) send(ms []raftpb.Message) {
 			ok, exceed := s.r.td.Observe(ms[i].To)
 			if !ok {
 				// TODO: limit request rate.
-				plog.Warningf("failed to send out heartbeat on time (deadline exceeded for %v)", exceed)
+				plog.Warningf("failed to send out heartbeat on time (exceeded the %v timeout for %v)", exceed, s.cfg.TickMs)
```

Comment:
switch exceed and s.cfg.TickMs. also add unit for the timeout in warningf (exceed the %vms timeout for %v) since tickms is an integer i assume.